distributed/ddp/example.py

```diff
@@ -1,11 +1,13 @@
 import argparse
 import os
+import tempfile
 import sys
 
 import torch
 import torch.distributed as dist
 import torch.nn as nn
 import torch.optim as optim
+from urllib.parse import urlparse
```

Comment:
looks like the convention of this file is that non-torch imports belongs to the previous import block

Reply:
thanks. updated